pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1844,7 +1851,7 @@ def _update_fstype_combo(self, device_type):
                 if active_index == btrfs_idx:
                     active_index = next(
                         idx for idx, data in enumerate(self._fsCombo.get_model())
-                        if data[1] == self.storage.default_fstype
+                        if data[0] == self._default_file_system
```

Comment:
I am just a bit confused here. We use row[1] a few lines above and also in _get_file_system_type. Shouldn't it be used (even the method) here as well?

Reply:
Good catch! It is a bug. Fixed.